README.md

```diff
@@ -1,5 +1,7 @@
 # HTML Project
 
```

Comment:
Nick, overall this is an effective project that shows me you have a comprehensive understanding of the basics or core knowledge of web dev in the browser.  Your layout is clear and easy to follow, and the purpose of your site is obvious and intuitive - UX/CX (Customer experience) is front and center.  

Two pushes - try to see if you can get your 'lighthouse score' (tool in the browser) to hit green when it looking at accessibility, your current score is 87.  It has some helpful suggestions to add to your page to make it even more accessible.

push two - have a look at your site on a mobile screen (use the browser inspector to do this).  In what ways could you use media queries and flexbox to make sure your UI maintains ease-of-viewing on smaller screens?